src/table/condensed.scss

```diff
@@ -4,7 +4,7 @@
 @import './variables';
 
 @mixin iui-table-condensed {
-  .iui-header .iui-cell,
+  .iui-table-header .iui-cell,
```

Comment:
Bringing back conversation about iui-header and the Website Header component vs iui-header the table header. I think iui-header & iui-body should be generic name. Header and Body can be in a modal, in a table, in tab. Header represent the container of the title, and body represent the container of the content. 
I think nested classes should be shorten while unnested class can use bigger naming. 
For the Header Component, we are talking about a site header, so I'm suggesting a renaming of this component. Could be iui-site-header, iui-main-header, iui-global-header. There is some Bentley's site using that header footer and the class are name : GlobalHeader, GlobalFooter. I'm suggesting that we adjust the Header component to GlobalHeader instead of adding middle name to nested class.